.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: ruby
+rvm:
+ - 2.2
```

Comment:
@davetoxa ok! got annoyed with the slow travis-ci builds so i just copy-pasted the default. :) 

does the ruby version need to match what i used in the application (2.4)? or it's not required for the build process?